### ISSUE TITLE
WCAG 2.0 validation

### ISSUE THREAD

**Issue by Floppy** [gjtorikian/html-proofer#366]
I found http://tenon.io the other day for validating sites against accessibility guidelines, which is great. It would be amazing if html-proofer could do this as part of a CI workflow. There are guidelines on accessibility evaluation at https://www.w3.org/WAI/eval/Overview, but I don't know if there is a specific set of pass/fail type tests for this stuff - I'm far from an expert...

**Comment by Floppy** [comment]
Just found https://github.com/accesslint/accesslint.rb, which might be able to take some of the load.

**Comment by Floppy** [comment]
And that links to a set of rules at https://github.com/GoogleChrome/accessibility-developer-tools/wiki/Audit-Rules.

**Comment by Floppy** [comment]
I could knock together a PR that uses `accesslint.rb` pretty quickly, but it depends on phantomjs being installed on the system. @gjtorikian is that the sort of thing you'd allow for optional checks? html-proofer is pretty pure, at the moment...

**Comment by gjtorikian** [comment]
I'll have to think about this a bit. On the one hand, I care about accessibility, and have hand-rolled or used other tooling to support this for CI. On the other hand, I added HTML validation to Proofer, and it ended up being a bad idea due to a bunch of false positives and work-arounds. I'd like to avoid that sort of situation here too, if it's possible. 😅

**Comment by Floppy** [comment]
Yeah, it's not really a hard-and-fast validation, so the potential for false positives is high. I know what you mean...

**Comment by fulldecent** [comment]
I recommend closing the issue as out of project scope (recommended scope discussed at #422).

This is still a great idea and can be implemented as a third-party module (and mentioned here on a wiki) if anyone is interested to work on it.

**Comment by fulldecent** [comment]
Closing this issue now based on the approved project scope.

I do also care about accessibility. Please see the brand new wiki pages which discuss how to extend html-proofer.

> https://github.com/gjtorikian/html-proofer/wiki

mailto-awesome in there is a simple example you can copy from to implement this additional check. I think a lot of people will want to use it and I would personally add it to my https://github.com/fulldecent/lightning-sites and https://github.com/fulldecent/html-website-template projects.
